deliveries/Archisinal-milestone_1.md

```diff
@@ -0,0 +1,41 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/smart-contract-bounty/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [Archisinal](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/Archisinal.md)
+
+**Context** (optional)
+> The 1st milestone focuses on the Smart Contracts part of our Artisanal MVP. It includes basic marketplace features like administering buyers and creators, selling NFTs with fixed prices or timed auctions, managing collections, etc.
+
+**Deliverables**
+
+| Number | Deliverable | Link                                                                                   | Notes                                                                                                                                                                                        |                             
+| ------------- | ------------- |----------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0a. | License | [LICENSE](https://github.com/archisinal/marketplace-contracts/blob/main/LICENSE)                   | MIT                                                                                                                                                                                          |
+| 0b. | Documentation | [README.md](https://github.com/archisinal/marketplace-contracts/blob/main/README.md)                      | Entry-point of docs is README.md, then use `cargo doc --open` to check all docs.                                                                                                             |
+| 0c. | Testing and Testing Guide | [TESTING.md](https://github.com/archisinal/marketplace-contracts/blob/main/TESTING.md)             | Created E2E, Security, Performance tests, and testing guide for them.                                                                                                                        |
```

Comment:
Your CI doesn't run all your tests, it only runs a small subset of them. For example, no `ink_e2e` tests or TS tests are run.

Also, your contracts don't get built either. I guess technically `cargo clippy` does do `cargo check` under the hood, but I'd be more explicit here.

Reply:
You can check TS tests log in the job `docker-test` and step `Run tests`. All 103 tests are passing: https://github.com/Archisinal/marketplace-contracts/actions/runs/6692253189/job/18181056276#step:7:1

I removed `ink_e2e` tests, as they were outdated. TS e2e tests fully covers the cases.

`cargo clippy` really runs `cargo check` under the hood. I renamed this job to `lint` to avoid confusions.

Also I added `Run build` step to be more explicit, when building contracts before running e2e tests.

Reply:
Okay gotcha, thanks for clarifying this!